src/io/leopard/toLeopard.ts

```diff
@@ -986,11 +980,15 @@ export default function toLeopard(
     for (const checkTarget of targetsToCheckForShowBlocks) {
       for (const script of checkTarget.scripts) {
         for (const block of script.blocks) {
-          if (block.opcode === OpCode.data_showvariable) {
-            shownWatchers.add(block.inputs.VARIABLE.value);
+          if (block.opcode === OpCode.data_showvariable || block.opcode === OpCode.data_hidevariable) {
+            shownWatchers.add(
+              (block as BlockBase<OpCode.data_showvariable, { VARIABLE: BlockInput.Variable }>).inputs.VARIABLE.value.id
+            );
           }
-          if (block.opcode === OpCode.data_showlist) {
-            shownWatchers.add(block.inputs.LIST.value);
+          if (block.opcode === OpCode.data_showlist || block.opcode === OpCode.data_hidelist) {
+            shownWatchers.add(
+              (block as BlockBase<OpCode.data_showlist, { LIST: BlockInput.List }>).inputs.LIST.value.id
+            );
```

Comment:
Thanks for the catch! There were errors here earlier during testing that I couldn't get TypeScript to avoid without this. I'm not sure what was causing it since this code does work without the assertions, as you said.